contents/copy/info/main.md

```diff
@@ -40,22 +40,26 @@ Eichenstraße 4, Berlin
 
 This year, for the first time ever, we have a Festival Area! We dedicated a part of our large and beautiful venue just to fun activities and short live demos that you can check out during breaks. You’ll be able to: meet the companies that support CSSconf EU, chat with the people behind open source projects / community initiatives, be wooed by { live : JS }, see the CSSconf bags getting printed, take selfies in our photo booth, play foosball, charge up your phones, grab a frozen yogurt and … sit on beanbags!
```

Comment:
from here on everything looks like in italics, is that just github?

Reply:
yeah, not sure why :/ it looks correct tho when rendered.